mm2src/coins/lightning/ln_sql.rs

```diff
@@ -824,12 +870,17 @@ impl LightningDB for SqliteLightningDB {
         preimage: PaymentPreimage,
     ) -> Result<(), Self::Error> {
         let for_coin = self.db_ticker.clone();
-        let payment_hash = hex::encode(hash.0);
         let preimage = hex::encode(preimage.0);
+        let last_updated = (now_ms() / 1000) as i64;
+        let payment_hash = hex::encode(hash.0);
 
         let sqlite_connection = self.sqlite_connection.clone();
         async_blocking(move || {
-            let params = [&preimage as &dyn ToSql, &payment_hash as &dyn ToSql];
+            let params = [
+                &preimage as &dyn ToSql,
+                &last_updated as &dyn ToSql,
+                &payment_hash as &dyn ToSql,
+            ];
```

Comment:
Please consider using the `rusqlite::params` or `db_common::owned_named_params` macros